pkgs/tools/networking/ndn-tools/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, ndn-cxx
+, openssl
+, doxygen
+, boost
+, sqlite
+, pkg-config
+, python3
+, python3Packages
+, wafHook
+, libpcap
+, fetchFromGitHub
+}:
+stdenv.mkDerivation rec {
+  pname = "ndn-tools";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "named-data";
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "1q2d0v8srqjbvigr570qw6ia0d9f88aj26ccyxkzjjwwqdx3y4fy";
+  };
+
+  nativeBuildInputs = [ pkg-config wafHook doxygen python3 python3Packages.sphinx ];
+  buildInputs = [ ndn-cxx boost libpcap openssl ];
+
+  wafConfigureFlags = [
+    "--boost-includes=${boost.dev}/include"
+    "--boost-libs=${boost.out}/lib"
+    # "--with-tests"
+  ];
+
+  # Upstream's tests don't all pass!
```

```suggestion
  # Upstream's tests don't all pass
```
Can we disable some?